### PR TITLE
refactor(metrics): unify storage_metric_level & streaming_metric_level to 'metrics_level'

### PR DESCRIPTION
**Pull request by fuyufjh**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Resolve https://github.com/risingwavelabs/risingwave/pull/11994#issuecomment-1702281148

As title. See `src/config/example.toml` 

by the way

- Fixed a bug in src/common/src/metrics/relabeled_metric.rs
- renamed "Bloom-Filter Miss Rate" to "Bloom-Filter Positive Rate"

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: src/risedevtool/src/task/compactor_service.rs [hunk 1]
```diff
@@ -55,7 +55,7 @@ impl CompactorService {
             .arg("--advertise-addr")
             .arg(format!("{}:{}", config.address, config.port))
             .arg("--metrics-level")
-            .arg("1");
```
Comment (zwang28) [inline src/risedevtool/src/task/compactor_service.rs:58]: Case-sensitive.

```
thread 'main' panicked at 'parse error TOML parse error at line 109, column 24
    |
109 | storage_metric_level = "info"
    |                        ^^^^^^
unknown variant `info`, expected one of `Disabled`, `Critical`, `Info`, `Debug`
```

File: src/risedevtool/src/task/compute_node_service.rs [hunk 2]
```diff
@@ -57,7 +57,7 @@ impl ComputeNodeService {
             .arg("--advertise-addr")
             .arg(format!("{}:{}", config.address, config.port))
             .arg("--metrics-level")
-            .arg("1")
```
Comment (zwang28) [inline src/risedevtool/src/task/compute_node_service.rs:60]: ditto

File: src/risedevtool/src/task/frontend_service.rs [hunk 3]
```diff
@@ -63,7 +63,7 @@ impl FrontendService {
                 config.listen_address, config.health_check_port
             ))
             .arg("--metrics-level")
-            .arg("1");
```
Comment (zwang28) [inline src/risedevtool/src/task/frontend_service.rs:66]: ditto

**Review by zwang28 — APPROVED** [review]
rest LGTM

File: src/common/src/metrics/relabeled_metric.rs [hunk 4]
```diff
@@ -76,7 +76,7 @@ impl RelabeledHistogramVec {
             for label in relabeled_vals.iter_mut().take(self.relabel_num) {
                 *label = "";
             }
-            self.metric.with_label_values(&relabeled_vals);
+            return self.metric.with_label_values(&relabeled_vals);
```
Comment (hzxa21) [inline src/common/src/metrics/relabeled_metric.rs:79]: Noticed this bug during debugging another issue

**Review by hzxa21 — APPROVED** [review]
LGTM

**Pull request merged**